cmd/collector/app/options.go

```diff
@@ -31,24 +31,27 @@ const (
 	DefaultQueueSize = 2000
 	// DefaultGRPCMaxReceiveMessageLength is the default max receivable message size for the gRPC Collector
 	DefaultGRPCMaxReceiveMessageLength = 4 * 1024 * 1024
+	// DefaultHandleDuplicateTagKey is the default behaviour of collector on receiving duplicate tag key
+	DefaultHandleDuplicateTagKey = "keep"
```

Comment:
I understand we want to be backwards compatible with original behaviour, though I'd like to understand, is the original behaviour considered correct or a bug?

Reply:
I would let maintainers to decide the default value. In my opinion, we should keep the current behaviour as default.